apis/kueue/v1beta1/provisioningrequestconfig_types.go

```diff
@@ -57,6 +57,53 @@ type ProvisioningRequestConfigSpec struct {
 	// +listType=set
 	// +kubebuilder:validation:MaxItems=100
 	ManagedResources []corev1.ResourceName `json:"managedResources,omitempty"`
+
+	// retryStrategy defines strategy for retrying ProvisioningRequest.
+	// If null, then the default configuration is applied. To switch off retry mechanism
+	//  set retryStrategy.backoffLimitCount to 0.
+	//
+	// +optional
+	RetryStrategy *ProvisioningRequestRetryStrategy `json:"retryStrategy,omitempty"`
+}
+
+var (
+	DefaultRetryStrategy = ProvisioningRequestRetryStrategy{
+		BackoffLimitCount:  nil,
```

Comment:
The default here was 3 (defaultMaxRetries), so I would suggest to stay with that.

Reply:
Fixed